charts/authentik/values.yaml

```
@@ -972,6 +972,9 @@ prometheus:
 postgresql:
   # -- enable the Bitnami PostgreSQL chart. Refer to https://github.com/bitnami/charts/blob/main/bitnami/postgresql/ for possible values.
   enabled: false
+  image:
+    repository: bitnami/postgresql
```

```suggestion
    repository: docker.io/bitnami/postgresql
```
just for the people that don't have docker.io as default repo

---

This is handled upstream: https://github.com/bitnami/charts/blob/main/bitnami/postgresql/values.yaml#L108